packages/@ourworldindata/grapher/src/mapCharts/WorldRegionsToProjection.ts

```diff
@@ -1,287 +1,18 @@
-    "Solomon Islands" = MapProjectionName.Oceania,
-    Somalia = MapProjectionName.Africa,
-    Somaliland = MapProjectionName.Africa,
-    "South Africa" = MapProjectionName.Africa,
-    "South Georgia and the South Sandwich Islands" = MapProjectionName.SouthAmerica,
-    "South Korea" = MapProjectionName.Asia,
-    "South Ossetia" = MapProjectionName.Asia,
-    "South Sudan" = MapProjectionName.Africa,
-    Spain = MapProjectionName.Europe,
-    "Sri Lanka" = MapProjectionName.Asia,
-    Sudan = MapProjectionName.Africa,
-    Suriname = MapProjectionName.SouthAmerica,
-    "Svalbard and Jan Mayen" = MapProjectionName.Europe,
-    Eswatini = MapProjectionName.Africa,
-    Sweden = MapProjectionName.Europe,
-    Switzerland = MapProjectionName.Europe,
-    Syria = MapProjectionName.Asia,
-    Taiwan = MapProjectionName.Asia,
-    Tajikistan = MapProjectionName.Asia,
-    Tanzania = MapProjectionName.Africa,
-    Thailand = MapProjectionName.Asia,
-    Timor = MapProjectionName.Asia,
-    Togo = MapProjectionName.Africa,
-    Tokelau = MapProjectionName.Oceania,
-    Tonga = MapProjectionName.Oceania,
-    Transnistria = MapProjectionName.Europe,
-    "Trinidad and Tobago" = MapProjectionName.NorthAmerica,
-    Tunisia = MapProjectionName.Africa,
-    Turkey = MapProjectionName.Asia,
-    Turkmenistan = MapProjectionName.Asia,
-    "Turks and Caicos Islands" = MapProjectionName.NorthAmerica,
-    Tuscany = MapProjectionName.Europe,
-    Tuvalu = MapProjectionName.Oceania,
-    "Two Sicilies" = MapProjectionName.Europe,
-    USSR = MapProjectionName.Europe,
-    Uganda = MapProjectionName.Africa,
-    Ukraine = MapProjectionName.Europe,
-    "United Arab Emirates" = MapProjectionName.Asia,
-    "United Kingdom" = MapProjectionName.Europe,
-    "United Korea" = MapProjectionName.Asia,
-    "United States" = MapProjectionName.NorthAmerica,
-    "United States Minor Outlying Islands" = MapProjectionName.Oceania,
-    "United States Virgin Islands" = MapProjectionName.NorthAmerica,
-    Uruguay = MapProjectionName.SouthAmerica,
-    Uzbekistan = MapProjectionName.Asia,
-    Vanuatu = MapProjectionName.Oceania,
-    Vatican = MapProjectionName.Europe,
-    Venezuela = MapProjectionName.SouthAmerica,
-    Vietnam = MapProjectionName.Asia,
-    "Wallis and Futuna" = MapProjectionName.Oceania,
-    "West Germany" = MapProjectionName.Europe,
-    "Western Sahara" = MapProjectionName.Africa,
-    Wuerttemburg = MapProjectionName.Europe,
-    Yemen = MapProjectionName.Asia,
-    "Yemen Arab Republic" = MapProjectionName.Asia,
-    "Yemen People's Republic" = MapProjectionName.Asia,
-    Yugoslavia = MapProjectionName.Europe,
-    Zambia = MapProjectionName.Africa,
-    Zanzibar = MapProjectionName.Africa,
-    Zimbabwe = MapProjectionName.Africa,
-    "Åland Islands" = MapProjectionName.Europe,
-}
+export const WorldRegionToProjection: Map<string, MapProjectionName> =
+    Object.fromEntries(
+        continents
+            .map(({ name, members }) =>
```

Comment:
nit: can use `flatMap` here directly, so we don't need the additional `.flat()` at the end